lib/evaluate/call.cc

```diff
@@ -111,6 +143,19 @@ const Symbol *ProcedureDesignator::GetSymbol() const {
       u);
 }
 
+parser::CharBlock ProcedureDesignator::GetName() const {
+  return std::visit(
+      common::visitors{
+          [](const SpecificIntrinsic &i) -> parser::CharBlock {
+            return i.name;
+          },
+          [](const Symbol *sym) -> parser::CharBlock { return sym->name(); },
+          [](const common::CopyableIndirection<Component> &c)
+              -> parser::CharBlock { return c.value().GetLastSymbol().name(); },
```

Comment:
I don't think the return type is necessary in the last two cases.

And in the first case, I think explicit construction (`return parser::CharBlock{i.name};`) is clearer, since the type has to be named somewhere in the lambda.

Reply:
Ok.